web/components/CaretakerTable.tsx

```diff
@@ -0,0 +1,165 @@
+                    {headerGroup.headers.map((column) => (
+                      <th
+                        scope="col"
+                        className="py-3 px-6 text-base font-normal tracking-wider text-slate-500 text-center"
+                        {...column.getHeaderProps()}
+                      >
+                        {column.render("Header")}
+                      </th>
+                    ))}
+                  </tr>
+                ))}
+              </thead>
+              <tbody {...getTableBodyProps()}>
+                {rows.map((row, i) => {
+                  prepareRow(row);
+                  return (
+                    <>
+                      <tr
+                        onClick={() => {
+                          setOpen((prevOpen) => {
+                            const newOpen = [...prevOpen];
+                            newOpen[i] = !newOpen[i];
+                            return newOpen;
+                          });
+                        }}
+                        className="cursor-pointer"
+                        {...row.getRowProps()}
+                      >
+                        <td className="border-t">
+                          <RiArrowDropDownLine
+                            className={`text-2xl duration-300 cursor-pointer ${
+                              open[i] && "rotate-180"
+                            }`}
+                          />
+                        </td>
+                        {row.cells.map((cell) => {
+                          return (
+                            <>
+                              <td
+                                className="py-4 px-6 text-base border-t font-normal text-black whitespace-nowrap"
+                                {...cell.getCellProps()}
+                              >
+                                {cell.column.id === "assigned" ? (
+                                  cell.value ? (
+                                    <div className="text-xs bg-blue-200 text-center rounded-md p-2">
+                                      Assigned
+                                    </div>
+                                  ) : (
+                                    <div className="text-xs bg-orange-200 text-center rounded-md p-2">
+                                      Not Assigned
+                                    </div>
+                                  )
+                                ) : (
+                                  cell.render("Cell")
+                                )}
+                              </td>
+                            </>
+                          );
+                        })}
+                        <td className="border-t">
+                          <div className="flex flex-row">
+                            <div
+                              className="pr-2 pt-1 cursor-pointer"
+                              onClick={() => {
+                                confirm(
```

Comment:
This should be a modal instead of a native "confirm" 😅 We can do that next sprint if you don't have time to make the modal just lmk 
![image](https://github.com/user-attachments/assets/f372cd23-3242-477b-b22a-0c1fffc87eac)